android/app/src/main/java/org/fossasia/openevent/core/track/session/SessionDetailActivity.java

```diff
@@ -112,6 +113,13 @@
     protected ImageView youtubeThumbnail;
     @BindView(R.id.watch)
     protected ImageButton playButton;
+    @BindView(R.id.sessionType)
+    protected TextView session_type;
+    @BindView(R.id.dot)
+    protected TextView dot;
+    @BindView(R.id.type)
+    protected TextView type_textview;
+
```

Comment:
That's not how you should do it. Define only one textview. If you need to hide it, wrap the layout in LinearLayout

Reply:
@iamareebjamal but if i do that the grid view will distort, the gridview has three columns in every row,
If i create two dummy textviews after the LinearLayout then the positioning of ':' will not be proper.
Either i should create a separate issue moving from grid layout to linear layout, as in the current code already there is a problem that when the track name is empty only ':' would be displayed.

Reply:
You can use framelayout as a wrapper. Check if that is possible